src/data/valid/Database-interleaved.yaml

```diff
@@ -1490,27 +1490,20 @@ biosample_set:
       has_raw_value: "9 gram per gram"
       has_unit: g/g
       type: nmdc:QuantityValue
-    add_recov_method:
-      type: nmdc:TextValue
-      has_raw_value: xxx
+    add_recov_method: Gas Injection;2023-05-15
     additional_info:
       type: nmdc:TextValue
       has_raw_value: xxx
-    address:
-      type: nmdc:TextValue
-      has_raw_value: xxx
+    address: 123 Main Street
```

Comment:
It's probably not a big deal, but I can't quite reconcile this change (and similar) with some of the comments I've read elsewhere. Specifically, `mixs-migration.md` says:

> The default NMDC pipeline converts all `string` → `TextValue`

Since I don't see `address` listed in `yq-for-mixs_subset_modified.txt` I'm sort of scratching my head about why this slot _doesn't_ still have a `TextValue` range.